pyiron_base/generic/datacontainer.py

```diff
@@ -10,6 +10,7 @@
 import warnings
 import numpy as np
 from .fileio import read, write
+from .hdfstub import HDFStub
```

Comment:
Personally I prefer global imports, or is there a specific reason to use relative imports here?

Reply:
Jup, will fix.